willow/plugins/wand.py

```diff
@@ -179,10 +179,30 @@ def save_as_webp(self, f, quality=80, lossless=False):
                     b"webp:lossless",
                     b"true",
                 )
+            else:
+                converted.compression_quality = quality
             converted.save(file=f)
 
         return WebPImageFile(f)
 
+    @Image.operation
+    def save_as_avif(self, f, quality=80, lossless=False):
+        with self.image.convert("avif") as converted:
+            converted.compression_quality = quality
+            if lossless:
```

Comment:
```suggestion
            if lossless:
```

since `converted.compression_quality = quality` is set in else ;)

Reply:
Oh man so silly of me!! alright now I understood it properly ; )